### PR TITLE
Fix Bug 1218905 - Because we :heart: the l10n community

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Comment by robhudson** [comment]
I ignored `W501: one character variable name` because they didn't seem so bad to me. But I could update those if anyone feels strongly about it.

**Comment by jezdez** [comment]
@robhudson I do feel strongly about them

**Comment by jezdez** [comment]
Yeah, assuming the variables names are fixed I'm r+

**Comment by groovecoder** [comment]
I've also started to replace 1-character variable names in my code and as I find it in the kuma code.

**Comment by jezdez** [comment]
@groovecoder 
giphy

File: kuma/wiki/search.py [hunk 1]
```diff
@@ -308,8 +308,8 @@ def reindex_all(cls, chunk_size=500, index=None, percent=100):
...
 
         message = _(
-            'Indexing {total} documents into {n} chunks of size {size} into '
-            'index {index}.'.format(total=total, n=total_chunks,
-                                    size=chunk_size,
-                                    index=index.prefixed_name))
+            'Indexing %(total)s documents into %(total_chunks)s chunks of '
+            'size %(size)s into index %(index)s.' % {
+                'total': total, 'total_chunks': total_chunks,
+                'size': chunk_size, 'index': index.prefixed_name})
```
Comment (robhudson) [inline kuma/wiki/search.py]: @jezdez can you spot check this? In an effort to move away from `.format` here while I'm at it.

**Comment by robhudson** [comment]
The only thing dennis complains about now is one of the strings from promote-mdn.

File: kuma/wiki/search.py (hunk 1 above)
Comment (jezdez) [inline kuma/wiki/search.py]: @robhudson Sure, here's the fixed version:

```python
message = _(
    'Indexing %(total)d documents into %(total_chunks)d chunks of '
    'size %(size)d into index %(index)s.' % {
        'total': total,
        'total_chunks': total_chunks,
        'size': chunk_size,
        'index': index.prefixed_name,
    }
)
```

Note the use of `%d` as the decimal integer representation.

File: kuma/demos/models.py [hunk 2]
```diff
@@ -201,8 +201,10 @@ def clean(self, *args, **kwargs):
         try:
             if file._size > self.max_upload_size:
                 raise ValidationError(
-                    _('Please keep filesize under %s. Current filesize %s') %
-                    (filesizeformat(self.max_upload_size), filesizeformat(file._size))
+                    _('Please keep filesize under %(max_size)s. '
+                      'Current filesize %(file_size)s') % {
+                          'max_size': filesizeformat(self.max_upload_size),
+                          'file_size': filesizeformat(file._size)}
```
Comment (jezdez) [inline kuma/demos/models.py]: Mind breaking before the closing curly braces here?

**Pull request merged**